app/templates/head.hbs

```diff
@@ -1 +1 @@
-{{head-tags headTags=model.headTags}}
+<HeadTags @headTags={{@model.headTags}} />
```

Comment:
https://github.com/ronco/ember-cli-head#head-data-service

<img width="869" alt="Screenshot 2020-09-29 at 15 41 00" src="https://user-images.githubusercontent.com/2526/94573655-34c44900-026a-11eb-832f-7429fcfff63e.png">

Should this be `this.model.headTags`?

Reply:
Certainly possible.  I'll make a PR

Reply:
nice one, thanks.

This issue might be related https://github.com/ronco/ember-cli-meta-tags/issues/90